conference.js

```diff
@@ -1459,6 +1475,19 @@ export default {
     useAudioStream(newStream) {
         return new Promise((resolve, reject) => {
             _replaceLocalAudioTrackQueue.enqueue(onFinish => {
+                const state = APP.store.getState();
+
+                // When the prejoin page is displayed localAudio is not set
+                // so just replace the audio track from the store with the new one.
+                if (isPrejoinPageVisible(state)) {
+                    const oldTrack = getLocalJitsiAudioTrack(state);
+
+                    return APP.store.dispatch(replaceLocalTrack(oldTrack, newStream))
```

Comment:
Tracks and streams usually mean something different. I know what you're doing is right, it just looks odd to replace a track with a stream.

Reply:
That's true. Do you think I should rename the conference methods (e.g from `useVideoStream` to `useVideoTrack`)? (Seems more appropriate)

Reply:
Renamed this to `oldTrack`. 

@zbettenbuk Could you please take another look?